contracts/wrappers/ccip/Router.ts

```diff
@@ -89,11 +90,17 @@ export class Router
     Contract
 {
   private ownable: ownable2step.ContractClient
+  readonly RMNOwnable: ownable2step.ContractClient
+
   constructor(
     readonly address: Address,
     readonly init?: { code: Cell; data: Cell },
   ) {
     this.ownable = new ownable2step.ContractClient(address)
+    this.RMNOwnable = new ownable2step.ContractClient(address, {
```

Comment:
Like this approach!

Should we also update and compose Go bindings in this PR? Tooling stack will need it.